src/functions/getTestResultsByTesterStaffId.ts

```diff
@@ -45,13 +45,13 @@ export const getTestResultsByTesterStaffId = async (event: any) => {
     if (event.queryStringParameters.testStatus) {
       filters.testStatus = event.queryStringParameters.testStatus;
     }
-    return testResultsService.getTestResults(filters)
+    return testResultsService.getTestResultsByTesterStaffId(filters)
```

Comment:
Can we flatten these out into await calls. We can then simplify the code in the called function

Reply:
will do in 18734